pages/_app.js

```diff
@@ -1,12 +1,87 @@
 import GlobalStyle from "../styles";
-import { bags } from "../components/List/data";
-import List from "./List";
+import { airlines } from "../lib/data";
+import { useState } from "react";
+import { calculateVolume } from "@/utils";
 
 export default function App({ Component, pageProps }) {
+  const [unitSystem, setUnitSystem] = useState("metric");
+  const [sortedAirlines, setSortedAirlines] = useState(airlines);
```

Comment:
calling the inital data initialAirlines frees up the name, so that the state can be called `airlines` which reads nicer in all of the following code imho.

```suggestion
  const [airlines, setAirlines] = useState(initialAirlines);
```

Of course you have to change `sortedAirlines` and `setSortedAirlines` everywhere

Reply:
Done as well.